docs/devcontainers/create-dev-container.md

```diff
@@ -184,7 +184,7 @@ When you make changes like installing new software, changes made in the Dockerfi
 In your Dockerfile, use `FROM` to designate the image, and the `RUN` instruction to install any software. You can use `&&` to string together multiple commands.
 
 ```docker
-FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-12
+FROM mcr.microsoft.com/devcontainers/javascript-node:0-18-bullseye
```

Comment:
```suggestion
FROM mcr.microsoft.com/devcontainers/javascript-node:0-18
```

`0-18` and `0-18-bullseye` correlate to the same image. To keep things simple, should we avoid using `-bullseye` or specific OS (wherever possible) ?

Reply:
Makes sense to me!